modules/theme-tailwind-magento/cypress/support/pageObjects/Checkout.js

```diff
@@ -54,4 +54,8 @@ export default class Checkout {
   getCustomerAddress() {
     return cy.get('[data-cy=customer-address]').first()
   }
+
+  checkOrderSummaryItems() {
```

Comment:
not sure about naming here

Reply:
I would rename it to getOrderSummaryItems